backend/routes/api.php

```diff
@@ -16,18 +17,27 @@
 |
 */
 
-Route::get('/categories', [CategoryController::class, 'index']);
-Route::get('/categories/{id}', [CategoryController::class, 'show']);
-Route::group(['middleware' => ['auth:sanctum', 'isAdmin']], function () {
-    Route::post('/categories/{id}', [CategoryController::class, 'update']);
-    Route::post('/category', [CategoryController::class, 'store']);
-    Route::delete('/categories/{id}', [CategoryController::class, 'destroy']);
-});
+Route::get('/service-categories', [ServiceCategoryController::class, 'index']);
```

Comment:
Do we need EndPoint to retrieve list of categories without  services? I`d like to send services with categories instead of  creating additional EndPoint.

Reply:
single responsibility, it is better to have separate endpoints